features/restaurant_owner.feature

```diff
@@ -4,7 +4,7 @@ Feature: As a restaurant Owner
 
   Scenario: I register as a Restaurant Owner
     Given there are "0" restaurant owners in the system
-    And I am on the "home" page
+    And I am on the "login" page
```

Comment:
So is the Restaurant registration not visible on every page? Was removing it a part of this feature?